signac/__main__.py

```diff
@@ -634,35 +634,36 @@ def main_update_cache(args):
         _print_err(f"Updated cache (size={n}).")
 
 
-# UNCOMMENT THE FOLLOWING BLOCK WHEN THE FIRST MIGRATION IS INTRODUCED.
-# def main_migrate(args):
-#     "Migrate the project's schema to the current schema version."
-#     from .contrib.migration import apply_migrations, _get_config_schema_version
-#     from packaging import version
-#     from .version import SCHEMA_VERSION
-#     project = get_project(_ignore_schema_version=True)
-#
-#     root = args.root_directory if args.root_directory else os.getcwd()
-#
-#     schema_version = version.parse(SCHEMA_VERSION)
-#     config_schema_version = _get_config_schema_version(root, schema_version)
-#
-#     if config_schema_version > schema_version:
-#         _print_err(
-#             "The schema version of the project ({}) is newer than the schema "
-#             "version supported by signac version {}: {}. Try updating signac.".format(
-#                 config_schema_version, __version__, schema_version))
-#     elif config_schema_version == schema_version:
-#         _print_err(
-#             "The schema version of the project ({}) is up to date. "
-#             "Nothing to do.".format(config_schema_version))
-#     elif args.yes or _query_yes_no(
-#         "Do you want to migrate this project's schema version from '{}' to '{}'? "
-#         "WARNING: THIS PROCESS IS IRREVERSIBLE!".format(
-#             config_schema_version, schema_version), 'no'):
-#         apply_migrations(root)
-#
-#
+def main_migrate(args):
+    """Migrate the project's schema to the current schema version."""
+    from .contrib.migration import _get_config_schema_version, apply_migrations
+    from .version import SCHEMA_VERSION
+
+    root = args.root_directory if args.root_directory else os.getcwd()
+
+    schema_version = int(SCHEMA_VERSION)
+    config_schema_version = _get_config_schema_version(root, schema_version)
+
+    if config_schema_version > schema_version:
+        _print_err(
+            f"The schema version of the project ({config_schema_version}) is "
+            "newer than the schema version supported by signac version "
+            f"{__version__}: {schema_version}. Try updating signac."
```

Comment:
I think I like this way better?
```suggestion
            f"newer than the schema version {schema_version} supported by "
            f"signac version {__version__}. Try updating signac."
```

Reply:
:+1: